doc/Mutable_References.md

```diff
@@ -1,13 +1,13 @@
 # Mutable References in Tock - Memory Containers
 
 Borrows are a critical part of the Rust language that help provide its
-safety guarantees. However, they can complicate event-driven code without  
+safety guarantees. However, they can complicate event-driven code without
 a heap (no dynamic allocation). Tock uses memory containers
 such as the TakeCell abstraction to allow simple code to keep
 the safety properties Rust provides.
 
-## Brief Overview of Borrowing in Rust 
-Ownership and Borrowing are two design features in Rust which 
+## Brief Overview of Borrowing in Rust
+Ownership and Borrowing are two design features in Rust which
 prevent race conditions and make it impossible to write code that produces
 dangling pointers.
 
```

Comment:
just below this (not in your changes, but I noticed)

"borrows makes" -> "borrows make"